docs/running.md

```diff
@@ -29,9 +29,17 @@ mkdir test-network
 ### Create a new Genesis file
 
 First, create a new `genesis.json` file devoid of accounts and validators. The `--base-fee` here is completely arbitrary.
+The `--power-scale` value of `0` means we'll grant 1 voting power per 1 FIL; to use more precision, we can set it to `3`
+to use milliFIL for example.
 
 ```shell
-cargo run -p fendermint_app -- genesis --genesis-file test-network/genesis.json new --chain-name test --base-fee 1000 --timestamp 1680101412
+cargo run -p fendermint_app -- \
+  genesis --genesis-file test-network/genesis.json \
+  new \
+  --chain-name test \
```

Comment:
I don't know if this is the case already, but I would force this one to be a `SubnetID` and make `chain_name == subnet_id` rejecting any `chain_name` that is not a valid `SubnetID`.

Reply:
IPC is optional with Fendermint; I think at the moment there is nothing to check that you are using a subnet ID for the name. IIRC the name is passed to CometBFT as-is so that it can use it in whatever way it wants, and within Fendermint it's effectively destroyed as soon as it's hashed into a number, we will never want to parse it. So, it's just a convention we agreed that we should use the subnet ID for the name because it's unique.